### PR TITLE
Update docs to clarify how to get plugin access

### PR DESCRIPTION
**Pull request by harrisonliew**
$subject

### CODE REVIEW

File: doc/Hammer-Basics/Hammer-Overview.rst [hunk 1]
```diff
@@ -35,13 +35,17 @@ The ``*.tech.json`` contains pointers to relevant PDK files and fundamental tech
...
 The TCL commands input to the tool are created using technology and design settings provided by the designer.
 
-There are already three Hammer tool plugin repos to which access may be granted to users of Hammer, pending approval from each tool vendor. These repos are ``hammer-cadence-plugins``, ``hammer-synopsys-plugins``, and ``hammer-mentor-plugins``. In them are tool plugin implementations for actions including synthesis, place-and-route, DRC, LVS, and simulation.
+There are already three Hammer tool plugin repos to which access may be granted to Hammer users. These repos are ``hammer-cadence-plugins``, ``hammer-synopsys-plugins``, and ``hammer-mentor-plugins``. In them are tool plugin implementations for actions including synthesis, place-and-route, DRC, LVS, and simulation.
+
+.. note:: If you are not a UCB BAR or BWRC affiliate and have access to tools from a specific vendor, please email hammer-plugins-access@lists.berkeley.edu with a request for which plugin(s) you would like read access to. There will be no support guarantee for the plugin repositories, but users are encouraged to file issues and contribute patches where needed.
```
Comment (edwardcwang) [inline doc/Hammer-Basics/Hammer-Overview.rst]: Maybe we should hold off on publicizing this until the list is confirmed to work? Don't  want to accidentally route inquiries to /dev/null temporarily

File: README.md [hunk 2]
```diff
@@ -3,23 +3,19 @@ Highly Agile Masks Made Effortlessly from RTL
...
 This is the master (unstable) development branch.
 
-Stable numbered releases can be found below:
-
-Releases: https://github.com/ucb-bar/hammer/releases
-
 Documentation
 =============
-Here are links to various pieces of documentation currently available:
+The main documentation is found at https://hammer-vlsi.readthedocs.io
```
Comment (edwardcwang) [inline README.md]: Should also link to the local `doc/` dir

File: doc/Hammer-Basics/Hammer-Overview.rst (hunk 1 above)
Comment (harrisonliew) [reply doc/Hammer-Basics/Hammer-Overview.rst]: Yes, this should have been draft until the list is up.

File: README.md (hunk 2 above)
Comment (harrisonliew) [reply README.md]: I disagree. `doc/` is just the sources for the readthedocs, so it's all the same info but harder to read...

Comment (edwardcwang) [reply README.md]: Hmm, good point. Maybe say something like sources are available in `doc/` for offline reading or something?

File: doc/Hammer-Basics/Hammer-Overview.rst (hunk 1 above)
Comment (edwardcwang) [inline doc/Hammer-Basics/Hammer-Overview.rst]:
```suggestion
.. note:: If you are not a UCB BAR or BWRC affiliate and have access to tools from a specific vendor, please email hammer-plugins-access@lists.berkeley.edu with a request for which plugin(s) you would like access to. There will be no support guarantee for the plugin repositories, but users are encouraged to file issues and contribute patches where needed.
```

**Pull request merged**